doc/how_to_build_macosx.md

```diff
@@ -23,7 +24,7 @@
 With homebrew, you can install them with following command.
 
 ```
-$ brew install glew lz4 libjpeg libpng lzo pkg-config libusb cmake git-lfs libmypaint
+$ brew install glew lz4 libjpeg libpng lzo pkg-config libusb cmake git-lfs libmypaint libtiff
```

Comment:
As commented [here](https://github.com/opentoonz/opentoonz/pull/2499#issuecomment-466259830) OT uses libtiff in `opentoonz/thirdparty/tiff-4.0.3/libtiff` . 
I tested force-uninstalling libtiff from Homebrew and confirmed that OT could be built successfully without it.